core/standard/clause_7_core.adoc

```diff
@@ -58,6 +58,9 @@ in the response are determined by the server based on parameters of the request.
 
 A `bbox` or `time` parameter may be used to select only a subset of the features in
 the collection (the features that are located in the bounding box or time period).
+If the collection does not contain bbox data it SHALL be considered to match all
```

Comment:
The statement is more specific, if we talk about features, not data in general. My suggestion:

"The `bbox` parameter SHALL be considered to match all features in the collection that are not associated with a location. The `time` parameter SHALL be considered to match all features in the collection that are not associated with a time stamp or period. "